Misc/NEWS.d/next/Library/2025-03-07-19-24-27.gh-issue-130959.xO8vVS.rst

```diff
@@ -0,0 +1,3 @@
+Fix pure Python implementation of ``time.fromisoformat()`` to reject times
```

Comment:
Please mention the module:

```suggestion
Fix pure Python implementation of ``datetime.time.fromisoformat()`` to reject times
```

Reply:
I don't see "datetime" in this NEWS entry. I just suggest to replace time.fromisoformat() with datetime.time.fromisoformat().

Reply:
See 4d444bec1595c5519781337952e7b9f4370800de.